### DISCUSSION TITLE
What countries in the world still use the title "Sultan" for their monarchs?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#735] [Galaxy Trivia] [answered]
*answer with flags*

**Comment by peterbaccaro** [comment, marked as answer]
🇧🇳  BRUNEI
🇴🇲  OMAN

**Comment by thuongtruong1009** [comment]
Brunei 🇧🇳
Oman 🇴🇲
Malaysia (specifically, the states of Johor, Kedah, Kelantan, and Terengganu) 🇲🇾

**Comment by SANKARAMDAS** [comment]
🇧🇳 BRUNEI
🇴🇲 Oman, are the only independent countries which retains the title "sultan" for their monarchs.

**Comment by techvmuppa** [comment]
As of my knowledge cutoff date of September 2021, there were only two countries in the world that still had a reigning sultan as their monarch: Brunei and Oman.

Brunei, officially known as the Nation of Brunei, the Abode of Peace, is a small country located on the island of Borneo in Southeast Asia. The current sultan of Brunei is Sultan Hassanal Bolkiah, who has been in power since 1967.

Oman, officially known as the Sultanate of Oman, is located in the Middle East on the southeastern coast of the Arabian Peninsula. The current sultan of Oman is Sultan Haitham bin Tariq Al Said, who ascended to the throne in 2020 following the death of his predecessor, Sultan Qaboos bin Said Al Said.

It's worth noting that there have been other countries in the past that had sultans as their monarchs, but they either abolished the title or transitioned to a different form of government.
